src/route-href.js

```diff
@@ -42,7 +42,13 @@ export class RouteHref {
         }
 
         let href = this.router.generate(this.route, this.params);
-        this.element.setAttribute(this.attribute, href);
+
+        if (this.element.au.controller) {
+          this.element.au.controller.viewModel.href = href;
+        } else {
+          this.element.setAttribute(this.attribute, href);
```

Comment:
Note how the value of `this.attribute` is set to the calculated href. This property is set to the value of "href" by default but can be changed in the binding by the developer in case they need to set a different attribute. We should use this on the view model as well, like this:

```
this.element.au.controller.viewModel[this.attribute] = href;
```

This will enable custom elements with other properties to be usable too.